code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -206,8 +206,15 @@
 			M.stuttering = 1
 		M.stuttering = min(M.stuttering+4, 10)
 		M.Dizzy(5)
-		if(iscultist(M) && prob(5))
-			M.say(pick("Av'te Nar'sie","Pa'lid Mors","INO INO ORA ANA","SAT ANA!","Daim'niodeis Arc'iai Le'eones","R'ge Na'sie","Diabo us Vo'iscum","Eld' Mon Nobis"))
+		if(iscultist(M) && prob(8))
+			switch(pick("speech", "message", "emote"))
+				if("speech")
+					M.say(pick("Av'te Nar'sie","Pa'lid Mors","INO INO ORA ANA","SAT ANA!","Daim'niodeis Arc'iai Le'eones","R'ge Na'sie","Diabo us Vo'iscum","Eld' Mon Nobis"))
+				if("message")
+					to_chat(M, "<span class='cult'>[pick("Shai'Tye-", "Dianu-", \
+					"They will not take you from me, Serva-", "Your blood is mine!")].</span>")
+				if("emote")
+					M.visible_message("<span class='warning'>[M] [pick("bleeds from the nose", "coughs up blackened blood", "thrashes wildly for a moment")].</span>")
```

Comment:
Not a fan of the coughing, since we normally use a sound for that. The others are fine.

Reply:
I can replace the verb with drools as it was originally, or spits or bleeds, maybe from the eyes.

Reply:
Your call, really.